src/content/docs/en/guides/cms/payload.mdx

```diff
@@ -6,7 +6,222 @@ stub: true
 service: Payload CMS
 ---
 
-[Payload](https://payloadcms.com/) is a headless CMS and application framework.
+[PayloadCMS](https://payloadcms.com/) is a headless open-source content management system that can be used to provide content for your Astro project.
+
+## Integrating with Astro
+
+## Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+2. **A MongoDB database** - PayloadCMS will ask you for a MongoDB connection string when creating a new project. You can set one up locally or use [MongoDBAtlas](https://www.mongodb.com/) to host a database on the web for free.
+3. **A PayloadCMS REST API** - Create a [PayloadCMS](https://payloadcms.com/) project and connect it to your MongoDB database during the setup.
+
+### Installing PayloadCMS
+
+Follow the [PayloadCMS Installation docs](https://payloadcms.com/docs/getting-started/installation) in order to install a fresh PayloadCMS app. During the PayloadCMS installation, you will be asked if you want to use a template.
+
+If you pick one of the available templates (such as 'blog'), additional collections will be generated for you. You can then skip the following section, because you already have a collection other than "Users" that you can use.
+
+If you don't pick any templates, only a single collection called "Users" will be generated.
+
+If you start the PayloadCMS server using `npm run dev`, you can go to `http://localhost:3000/admin` to enter your PayloadCMS Dashboard for the first time. Create the initial administrator account and enter the Dashboard. You should now see that the "Users" collection contains just the administrator account.
```

Comment:
```suggestion
1. Follow the [PayloadCMS Installation docs](https://payloadcms.com/docs/getting-started/installation) in order to install a fresh PayloadCMS app. 

:::note[Choosing a template]
During the PayloadCMS installation, you will be asked if you want to use a template.

If you do not choose a template, only a single collection called "Users" will be generated and you will have to [define a PayloadCMS collection manually](#creating-a-payload-cms-collection) in Astro to use for your content.

Choosing any of the available templates at this step (such as 'blog') automatically generates additional collections for you to use.
:::

2. Start the PayloadCMS server using `npm run dev` and visit `http://localhost:3000/admin` to enter your PayloadCMS Dashboard for the first time. 

3. Using the Dashboard, create an initial administrator account, which should now be visible as the only entry in your "Users" collection.
```

Reply:
one thing to note here is that the cms will not start if a valid mongo connection is not established. Maybe it will be a good thing to expand a bit on how to create a new mongo instance locally or using Atlas since this blocks the following steps

Reply:
> one thing to note here is that the cms will not start if a valid mongo connection is not established. Maybe it will be a good thing to expand a bit on how to create a new mongo instance locally or using Atlas since this blocks the following steps

This is explained in the PayloadCMS docs. I can expand on it here as well if needed, of course.